native/package.json

```diff
@@ -42,14 +42,14 @@
     "@sentry/react-native": "^1.7.2",
     "@testing-library/jest-native": "^3.4.2",
     "@testing-library/react-native": "^7.0.2",
-    "api-client": "file:../api-client",
-    "build-configs": "file:../build-configs",
+    "api-client": "7.0.0",
+    "build-configs": "2.0.0",
     "hashids": "^2.2.1",
     "htmlparser2": "^4.0.0",
     "i18next": "^19.1.0",
     "is-url": "^1.2.4",
     "js-md5": "^0.7.3",
-    "locales": "file:../locales",
+    "locales": "1.0.0",
```

Comment:
I did this task within here as I already started with it: https://issues.tuerantuer.org/browse/IGAPP-366

Reply:
Do we still have to bump versions? Otherwise it's probably better to just set all monorepo package versions to `0.0.1` or something.